imports/plugins/core/graphql/lib/hocs/withPrimaryShopId.test.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { compose } from "recompose";
+import { mount } from "enzyme";
+import { MockedProvider } from "react-apollo/test-utils";
+import getPrimaryShopId from "../queries/getPrimaryShopId";
+import withPrimaryShopId from "./withPrimaryShopId";
+
+const fakeOpaqueShopId = "cmVhY3Rpb24vc2hvcDpKOEJocTN1VHRkZ3daeDNyeg==";
+const MockComponent = () => <div>Mock</div>;
+const TestComponent = compose(withPrimaryShopId)(MockComponent);
```

Comment:
Is using `compose` necessary in this case when you could just do `withPrimaryShopId(MockComponent)`?

Reply:
Good catch, it isn't